miniscope_io/models/stream.py

```diff
@@ -85,6 +96,22 @@ class StreamDaqConfig(MiniscopeConfig, YAMLMixin):
     block_size: int
     num_buffers: int
     LSB: Optional[bool]
+    save_video: Optional[bool] = False
+    save_video_filename: Optional[str] = 'output'
+    show_video: Optional[bool] = True
+    test_raw_data_path: Optional[Path] = TESTDATA_DIR / 'stream_daq_test_fpga_raw_input_200px.bin'
+
+    @field_validator('mode', mode='before')
+    def check_mode_string(cls, value: str) -> str:
+        if value not in ALLOWED_MODES:
+            value = 'DAQ'
+        return value
+
+    @field_validator('save_video_filename', mode='before')
+    def check_video_filename_string(cls, value: str, values) -> str:
```

Comment:
this looks like it will cause the model to fail to instantiate with default values, since `save_video = False` and `save_video_filename = output`